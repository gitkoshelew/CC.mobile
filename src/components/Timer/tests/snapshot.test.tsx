import {render} from '@testing-library/react-native';
import {Timer, IProps} from '../index';

const TimerProps = {
  timeInMinutes: '10',
  timeInSeconds: '00',
} as IProps;

it('timer snapshot', (): void => {
  const timer = render(<Timer {...TimerProps} />);

  expect(timer).toMatchSnapshot();
});
