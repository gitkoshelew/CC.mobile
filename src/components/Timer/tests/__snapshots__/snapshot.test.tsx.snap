// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`timer snapshot 1`] = `
<View
  style={
    [
      {
        "display": "flex",
        "flexDirection": "row",
        "justifyContent": "center",
      },
    ]
  }
>
  <Text
    style={
      [
        {
          "color": "#000000",
          "fontFamily": "Montserrat-Regular",
          "fontSize": 30,
        },
      ]
    }
  >
    NaN
  </Text>
  <Text
    style={
      [
        {
          "color": "#000000",
          "fontFamily": "Montserrat-Regular",
          "fontSize": 30,
        },
      ]
    }
  >
     : 
  </Text>
  <Text
    style={
      [
        {
          "color": "#000000",
          "fontFamily": "Montserrat-Regular",
          "fontSize": 30,
        },
      ]
    }
  >
    NaN
  </Text>
</View>
`;
